assets/tests/components/notificationModals/chelseaLoweredNotificationModal.test.tsx

```diff
@@ -1,10 +1,18 @@
 import { jest, describe, test, expect } from "@jest/globals"
-import React from "react"
+import React, { ReactNode, ReactPortal } from "react"
 import { render } from "@testing-library/react"
 import userEvent from "@testing-library/user-event"
 import { StateDispatchProvider } from "../../../src/contexts/stateDispatchContext"
 import { initialState, setNotification } from "../../../src/state"
 import ChelseaLoweredNotificationModal from "../../../src/components/notificationModals/chelseaLoweredNotificationModal"
+import ReactDOM from "react-dom"
+
+jest.mock("react-dom", () => {
+  return {
+    ...(jest.requireActual("react-dom") as typeof ReactDOM),
+    createPortal: (node: ReactNode): ReactPortal => node as ReactPortal,
+  }
+})
```

Comment:
question: what happens without this mock?

Reply:
Hmm, seems the issue is related to our use of `import renderer from "react-test-renderer"` in some of these tests?
It doesn't seem like all of the tests which had this added need it.

[undocumented :c] We're trying to move off `react-test-renderer` fwiw, so if you wanted to go ahead and do that conversion I'd be in support of that. Happy to pair if you'd like. I can also fix this if you'd prefer.

Reply:
Oh good catch! I missed that the chelseaLoweredNotificationModal didn't need it. Removed it there, and now I'm looking into what moving off `react-test-renderer` in the other 3 files will entail.

Reply:
for posterity, this is the `render` method we've preferred to use recently `import { render } from "@testing-library/react"`

to get snapshots to work with this, you'll likely need to use `asFragment()` or `baseElement` as the return type is different between the two render methods.

Reply:
Ok so. I did test out using render from "@testing-library/react", but ran into a snag, covered in a SO issue [here](https://stackoverflow.com/questions/69240360/react-jest-snapshot-only-returning-empty-documentfragment). Basically, the usually `asFragment` mechanics don't play nicely with modals, which are rendered separately from the render functions container. So I'm trying out a couple things to figure out if this is something to do now or in another task. I'm happy to do it, but just trying to figure out scope

Reply:
Ah, if the content of `asFragment` doesn't have any of the modal in it, then you'd likely want `baseElement` which points to the `<body>` iirc.

Reply:
Oh I forgot to make the same changes with modal. Might as well!